kaiax/gasless/impl/getter.go

```diff
@@ -185,6 +186,11 @@ func (g *GaslessModule) IsExecutable(approveTxOrNil, swapTx *types.Transaction)
 		if !ok || !g.isApproveTx(approveArgs) {
 			return false
 		}
+
+		if swapTx.Nonce() != approveTxOrNil.Nonce()+1 {
+			return false
+		}
```

Comment:
Let's move it below SP2, and call it SP3. If the KIP changes https://github.com/kaiachain/kips/pull/47/files#r1967130975.